### PR TITLE
(MAINT) added archiving SUT files doc

### PR DESCRIPTION
**Pull request by kevpl**
[skip ci]

@nwolfe what do you think? @puppetlabs/beaker?

### CODE REVIEW

File: docs/how_to/archive_sut_files.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+If you're preserving hosts just to SSH in and look at log files, however, this
+can be a tedious exercise. Why not just bring the log files to you on the
+beaker coordinator? This doc explains exactly how to do that using our
+`archive_file_from` Domain-Specific Language (DSL) method
+([method rubydocs](http://www.rubydoc.info/github/puppetlabs/beaker/Beaker/DSL/Helpers/HostHelpers#archive_file_from-instance_method)).
+
+# How Do I Use This?
+
+`archive_file_from` is a part of the beaker DSL, so its available in all test
```
Comment (nwolfe) [inline docs/how_to/archive_sut_files.md]: s/its/it's

**Comment by nwolfe** [comment]
👍 

We might as well link to this from the Confluence page once it's merged too.

**Comment by nwolfe** [comment]
Also, maybe we should refer to this document from `preserve_hosts.md` too

**Comment by kevpl** [comment]
@nwolfe agreed on referencing it from the confluence page. I don't think it needs to remove any text from that page though, it should probably just be in a links or references section somewhere.

Feedback included. What do you think?

**Comment by nwolfe** [comment]
> I don't think it needs to remove any text from that page though, it should probably just be in a links or references section somewhere.

Exactly, just some sort of "Additional References" section at the bottom or something

**Comment by nwolfe** [comment]
👍

File: docs/how_to/archive_sut_files.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# How to Archive Files from the Systems Under Test (SUTs)
+
+Oftentimes when you're dealing with beaker test development, or troubleshooting
```
Comment (tvpartytonight) [inline docs/how_to/archive_sut_files.md]: one too many

**Comment by kevpl** [comment]
@tvpartytonight any other comma issues / others?

**Comment by tvpartytonight** [comment]
👍

**Pull request merged**